src/sagas/options.js

```diff
@@ -0,0 +1,228 @@
+// @flow
+
+import Api from '_/ovirtapi'
+import { all, put, select, takeLatest, takeEvery, call } from 'redux-saga/effects'
+
+import { saveSSHKey as saveSSHKeyAction, saveUserOptionsOnBackend, setOption, setSSHKey, loadingUserOptionsInProgress, loadingUserOptionsFinished, stopSchedulerForResumingNotifications, startSchedulerForResumingNotifications } from '_/actions'
```

Comment:
I've been converting over to just import all actions to a var and referencing them that way....

```js
import * as A from '_/actions'

...

A.saveSSHKey(...)
```

that way we don't need to rename action functions etc.